history/cultures/dragonmaw.txt

```diff
@@ -1,25 +1,28 @@
 1.1.1 = {
 	discover_innovation = innovation_mustering_grounds
 	#
-	discover_innovation = innovation_casus_belli 
-	#
-	discover_innovation = innovation_taming_the_skies
-	discover_innovation = innovation_flying_beasts
+	discover_innovation = innovation_casus_belli
 }
 270.1.1 = {
  	discover_innovation = innovation_barracks 
 	#
 	discover_innovation = innovation_plenary_assemblies 
+	#
+	discover_innovation = innovation_wolfrider
 }
 400.1.1 = {
 	discover_innovation = innovation_motte 
 	#
 	discover_innovation = innovation_city_planning
+	#
+	discover_innovation = innovation_dragon
```

Comment:
They have dragons but see the Flying Beast tech.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/138373922-90bf7335-d6c2-45f9-9d72-3d4c3f01728c.png)

</details>